pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -317,8 +317,14 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		if len(options.DNSIP) > 0 {
```

Comment:
if we don't start the dns server unless DNSBindAddress is specified, what's the point of setting it to DNSIP initially?